jdbc-adapter/virtualschema-common/src/main/java/com/exasol/adapter/sql/SqlSelectList.java

```diff
@@ -17,7 +17,7 @@
     /**
      * Call this if all columns are required, i.e. SELECT * FROM ...
      */
-    public SqlSelectList() { }
+    public SqlSelectList() {super(null);}
```

Comment:
We could always create an empty arraylist so that we never have to deal with null handling (probably a bit safer). Also we could change this to factory methods.